src/pages/workspace/tags/WorkspaceViewTagsPage.tsx

```diff
@@ -234,6 +234,18 @@ function WorkspaceViewTagsPage({route}: WorkspaceViewTagsProps) {
                     cancelText={translate('common.cancel')}
                     danger
                 />
+                <View style={[styles.pv4, styles.ph5]}>
+                    <ToggleSettingOptionRow
```

Comment:
Coming from this issue https://github.com/Expensify/App/issues/43758, we should have hide the required option for dependent tags, because dependent tag lists cannot be required. More context https://github.com/Expensify/App/issues/43758#issuecomment-2167952431 https://github.com/Expensify/App/issues/43758#issuecomment-2168010545